presto-raptor/src/test/java/com/facebook/presto/raptor/storage/StorageOrganizationUtil.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.raptor.storage;
+
+import com.facebook.presto.raptor.backup.BackupStore;
+import com.facebook.presto.spi.HostAddress;
+import com.facebook.presto.spi.Node;
+import com.facebook.presto.spi.NodeManager;
+import com.facebook.presto.spi.NodeState;
+import com.google.common.collect.ImmutableSet;
+
+import java.io.File;
+import java.net.URI;
+import java.util.Set;
+import java.util.UUID;
+
+import static java.util.Objects.requireNonNull;
+
+public class StorageOrganizationUtil
+{
+    private StorageOrganizationUtil()
+    {}
+
+    public static NodeManager createNodeManager(String current, String... others)
+    {
+        Node currentNode = new TestingNode(current);
+
+        ImmutableSet.Builder<Node> nodes = ImmutableSet.builder();
+        nodes.add(currentNode);
+        for (String other : others) {
+            nodes.add(new TestingNode(other));
+        }
+
+        return new TestingNodeManager(nodes.build(), currentNode);
+    }
+
+    public static class TestingNodeManager
```

Comment:
You should rebase this branch. I just deleted `TestingNodeManager` and `TestingNode`